### PR TITLE
13.0 new api

### PR DESCRIPTION
**Pull request by sebastienbeau**

### CODE REVIEW

**Review by PierrickBrun — APPROVED** [review]

File: test-requirements.txt [hunk 1]
```diff
@@ -8,3 +8,6 @@ vcrpy-unittest
 openupgradelib @ git+https://github.com/OCA/openupgradelib.git
 unittest2 # For shopinvader test_controller, which inherits component
 odoo-test-helper
+
+odoo13-addon-base-rest @ git+https://github.com/OCA/rest-framework@refs/pull/71/head#subdirectory=setup/base_rest
+odoo13-addon-datamodel @ git+https://github.com/OCA/rest-framework@refs/pull/71/head#subdirectory=setup/datamodel
```
Comment (lmignon) [inline test-requirements.txt]: @sebastienbeau datamodel is no more required. (datamodel bindings is provided by a dedicated addon now)

Comment (PierrickBrun) [reply test-requirements.txt]: Thansk @lmignon , I did the change

File: shopinvader/tests/common.py [hunk 2]
```diff
@@ -30,14 +30,16 @@ def _install_lang_odoo(env, lang_xml_id, full_install=False):
...
-class CommonMixin(ComponentMixin):
+class CommonMixin(RegistryMixin, ComponentMixin):
     @staticmethod
     def _setup_backend(cls):
         cls.env = cls.env(context={"lang": "en_US"})
         cls.backend = cls.env.ref("shopinvader.backend_1")
         cls.backend.bind_all_product()
         cls.shopinvader_session = {}
         cls.existing_jobs = cls.env["queue.job"].browse()
+        cls.setUpComponent()
```
Comment (simahawk) [inline shopinvader/tests/common.py]: these should stay in setUpClass IMO

File: shopinvader/tests/common.py [hunk 3]
```diff
@@ -121,7 +123,6 @@ def setUpClass(cls):
             )
         )
         CommonMixin._setup_backend(cls)
-        cls.setUpComponent()
```
Comment (simahawk) [inline shopinvader/tests/common.py:124]: here

File: shopinvader/tests/common.py (hunk 2 above)
Comment (PierrickBrun) [reply shopinvader/tests/common.py]: Done

**Pull request closed without merging**